depobs/worker/tasks.py

```diff
@@ -258,6 +259,9 @@ def score_package(
         )
 
     pr.scoring_date = datetime.datetime.now()
+    pr.status = "scanned"
+
+    store_package_report(pr)
```

Comment:
@g-k this was there before but it looks like its no longer there - should I remove it?

Reply:
yeah we can drop it since I'm aggregating all the reports and saving them at the end now.